### PR TITLE
Pagination Alignment Changes

### PR DESCRIPTION
**Pull request by Git-Hub-Nanda**
- Pagination section aligned to center of the page.
- Added scroll bar to the table data.

### CODE REVIEW

File: products/src/components/ProductList.js [hunk 1]
```diff
@@ -53,8 +53,9 @@ class ProductList extends React.Component {
                 <div className='container-fluid'>
                     <h3 className='p-2'>Products</h3>
                     <Link to="/add"><Button varient="outline-secondary" size="sm">Add Product</Button></Link>
-                    <Table hover>
-                        <thead>
+                    <div style={{'height':'72vh','overflow-y':'scroll'}}>
```
Comment (git-hub-raja) [inline products/src/components/ProductList.js]: Please move the inline css to a file and import and use it.

File: products/src/components/ProductList.js [hunk 2]
```diff
@@ -53,8 +53,9 @@ class ProductList extends React.Component {
...
+                    <Table hover >
+                        <thead style={{'position':'sticky','top':'0'}}>
```
Comment (git-hub-raja) [inline products/src/components/ProductList.js]: Please move this inline css also to a separate file.

File: products/src/components/utils/paginate.js [hunk 3]
```diff
@@ -58,7 +58,7 @@ const Paginate = ({totalRecords, recordsPerPage = 10, siblingPageCount = 5, load
     }
     return(
         <>
-            <Pagination>
+            <Pagination style = {{'justify-content':'center','margin': '20px'}}>
```
Comment (git-hub-raja) [inline products/src/components/utils/paginate.js]: Please move this inline css also to a separate file.

**Review by git-hub-raja — CHANGES_REQUESTED** [review]
Hi @Git-Hub-Nanda ,

Please address the css changes.

Thank you.

File: products/src/components/ProductList.js (hunk 1 above)
Comment (Git-Hub-Nanda) [reply products/src/components/ProductList.js]: Ok. I will address.

File: products/src/css/common.css
Comment (git-hub-raja) [inline products/src/css/common.css:1]: Move these css to index.css and use. Get rid of this file and related code changes.

**Review by git-hub-raja — APPROVED** [review]
Looks Good.

**Comment by git-hub-raja** [comment]
This PR can be merged now.

**Pull request closed without merging**